website/src/content/service-contract.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Service Contract
+---
+
+<Subtitle>
+
+This page describes the different aspects of the service contract between commercetools and consumers of the custom applications features.
+
+</Subtitle>
+
+# Beta
+
+Beta features are indicated with the `BETA` flag. The flag can be visible in the main navigation of the documentation website as well as at the top of a page. The latter is a link to this page.
+
+A feature is marked as Beta when is stable enough to be used in production but is not fully feature complete.
+During the Beta period there might still be issues and planned breaking changes. In this phase commercetools will gather the necessary feedback from consumers of the features in order to finalize the missing things and promote the feature to general availability.
+
+# Hosting
+
+Customers using custom applications are responsible for the maintainance of the their custom applications. This includes hosting and deployments.
```

Comment:
"maintainance of the their custom applications"

maintenance is spelled with e. Find and replace to fix all occurrences.

The 'the' in "**the** their" is superfluous.